liquibase-core/src/main/java/liquibase/sqlgenerator/core/CreateViewGenerator.java

```diff
@@ -93,7 +105,15 @@ public ValidationErrors validate(CreateViewStatement createViewStatement, Databa
                 }
             }
         }
+
+        if (mssqlSetStatementsBefore != null && !mssqlSetStatementsBefore.isEmpty()) {
```

Comment:
```suggestion
        if (CollectionUtils.isNotEmpty(mssqlSetStatementsBefore)) {
```

Reply:
Will work in case if liquibase has apache commons collections dependency.

Reply:
Unfortunately Liquibase does not have this dependency :(